pkg-manager/core/test/catalogs.ts

```diff
@@ -902,6 +907,7 @@ describe('add', () => {
     })
     expect(readLockfile()).toMatchObject({
       catalogs: { default: { 'is-positive': { specifier: '1.0.0', version: '1.0.0' } } },
+      importers: { project1: { dependencies: { 'is-positive': { specifier: 'catalog:', version: '1.0.0' } } } },
```

Comment:
Before passing in `dir`, adding this line results in a test failure:

<img width="842" alt="Screenshot 2025-05-10 at 2 15 22 PM" src="https://github.com/user-attachments/assets/fb2f41bf-df65-45c4-9cfb-9d31c8b98dd9" />